### PR TITLE
ES-57 Add healthy checks and proper depends_on

### PR DESCRIPTION
**Pull request by heidrifx**
- added docker compose healthy checks
- updated docker compose depends_on relations
- is the weird behaviour from Rene appearing on any other machine? (docker compose waits for one service (RabbitMQ) which then ultimately errors out)
  - -> this should be fixed before merging!

### CODE REVIEW

**Comment by Knieboike** [comment]
image
- Tested on Apple M2 Pro

**Comment by heidrifx** [comment]
Fixed Rene's problem by increasing the timeout and retries of the analysismanger's healthy check

File: docker-compose.yml [hunk 1]
```diff
@@ -59,16 +73,24 @@ services:
...
+    healthcheck:
+      test: ["CMD", "pg_isready", "-U", "tad"]
+      interval: 10s
+      timeout: 5s
+      retries: 3
 
   # Core Services
   analysismanager:
-    image: well5a/analysis-manager:latest
+    image: well5a/analysis-manager:testing
```
Comment (eliasmueller) [inline docker-compose.yml:84]: should we really use the testing images here in the main branch?

Comment (heidrifx) [reply docker-compose.yml:84]: sadly, yeah, as the dev branches of our repos (tagged with testing here) are way ahead of the main branch and currently are the only ones including `curl`

Comment (eliasmueller) [reply docker-compose.yml:84]: - hmm okay maybe we should merge dev in main then in all other branches :D but well - don't take it as a blocker here

Comment (heidrifx) [reply docker-compose.yml:84]: yeah, I'd much rather prefer using the latest (i.e., main branch) tag but like I mentioned, the required `curl` command for the healthy checks are currently only available in the testing images

File: docker-compose.yml [hunk 2]
```diff
@@ -18,11 +17,16 @@ services:
     volumes:
     #  - ./rabbitmq/etc/definitions.json:/etc/rabbitmq/definitions.json
```
Comment (365Bit) [inline docker-compose.yml:18]: This part isn't necessary anymore - could we also remove these comments?

File: docker-compose.yml (hunk 1 above)
Comment (heidrifx) [reply docker-compose.yml:84]: @eliasmueller just as an FYI: we talked about it in the planning for this sprint and delayed this change until the last two week sprint
-> I'll merge this PR later and this **must** be done in the last two weeks of the project!

**Pull request merged**